DA/Internal/Desugar.hs

```diff
@@ -154,10 +154,10 @@ data RequiresT a b = RequiresT
 
 class HasMethod i (m : Symbol) r | i m -> r
 
-newtype Method t i (m : Symbol) r = Method { unMethod : t -> r }
+newtype Method t i (m : Symbol) = Method ()
```

Comment:
This means that a `Method` value no longer contains the method implementation. This is okay because we don't use the value at runtime